composer.json

```diff
@@ -17,9 +17,9 @@
         "install-bats": "if [ ! -f tools/bin/bats ] ; then git clone https://github.com/sstephenson/bats.git tools/bats; tools/bats/install.sh tools; fi",
         "install-phpcs": "mkdir -p tools/phpcs && cd tools/phpcs && COMPOSER_BIN_DIR=../../vendor/bin composer require squizlabs/php_codesniffer:^2.7",
         "install-phpunit": "mkdir -p tools/phpunit && cd tools/phpunit && COMPOSER_BIN_DIR=../../vendor/bin composer require phpunit/phpunit:^6",
-        "functional": "TERMINUS_PLUGINS_DIR=.. PATH=tools/bin:$PATH bats tests/terminus-1",
-        "cs": "PATH=tools/bin:$PATH phpcs --standard=PSR2 -n src Commands",
-        "cbf": "PATH=tools/bin:$PATH phpcbf --standard=PSR2 -n src Commands",
+        "functional": "PATH=tools/bin:$PATH bats tests/functional",
+        "cs": "PATH=vendor/bin:$PATH phpcs --standard=PSR2 -n src Commands",
```

Comment:
If you want to move the tools to vendor/bin, then you will also need to add them to `require-dev`. However, this won't work for bats. If we want to remove the "install tools" step, maybe we should switch to phpunit-based testing.

Reply:
This is actually correcting a bug.  The phpcs and phpcbf executables are located in `vendor/bin`, not `tools/bin`.  It might make sense to remove the `PATH=...` line and just call `vendor/bin/phpcs` directly.

Reply:
`require-dev` is empty in this composer.json, ergo, phpcs et. al. probably won't be found there.

The design / setup of the tests were intended to allow the tests to be run without installing any dev dependencies into the `vendor` directory. This was done due to the limitations of Terminus 2, where dev dependencies would often conflict with Terminus' own dependencies. The way the plugin avoided dev dependencies was via the `install-bats` commands & etc., which installs them to `tools`. Setting the PATH and then calling `bats` allowed the tests to be run in environments that had a global `bats` and an empty tool directory, or in environments that used the install-bats script to install it to tools.

Since Terminus 3 no longer has the restriction on dev dependencies, we could do as you suggest and just call `vendor/bin/phpcs` directly. However, to do that you'd have to add phpcs to `require-dev`. Also, bats can't be installed via Composer. I suppose that you could keep the `install-bats` trick and use `require-dev` for just the PHP tools if you wanted.

It would be a good idea to turn on testing for GitHub actions, so that we don't need to worry about the Circle permissions. It wouldn't be hard to make Circle run again, but GitHub actions are probably a better example for open source repositories.

Reply:
Thank you for the feedback.  I'm a bit confused.  Doesn't `COMPOSER_BIN_DIR=../../vendor/bin` cause phpcs to be installed in the vendor directory?  This is what I experienced in my testing.  If you execute `composer install-tools` then `vendor/bin/phpcs` works, hence to proposed change to fix.  What are you thoughts regarding the new Testing section in README.md?

Reply:
@greg-1-anderson: How should we proceed with this PR?  The most recent merge has created a merge conflict.  Should I just close?